Documentation/Metrics/rocksdb_background_errors.yaml

```diff
@@ -1,7 +1,7 @@
 name: rocksdb_background_errors
 introducedIn: "3.6"
 help: |
-  rocksdb_background_errors
+  RocksDB metric "background-errors"
 unit: number
 type: gauge
```

Comment:
Wondering if this should be of type "counter" if it shows the accumulated number.

Reply:
We basically only query values from RocksDB and then set the gauge. In such a case I prefer a gauge. Furthermore, errors should be rare, so that the total value is probably a better visualization than a rate of increase, which would be the default for a counter. I would leave it. Furthermore, our mechanism for RocksDB metrics simply exports everything as gauges and we would have to build infrastructure to make some into counters.

Reply:
Fine for me too.